Backend/app/rider-platform/rider-app/Main/spec/Storage/MultiModal.yaml

```
@@ -55,6 +62,7 @@ Journey:
     journeyExpiryTime: Maybe UTCTime
     merchantId: Id Merchant
     merchantOperatingCityId: Id MerchantOperatingCity
+    isNormalRideJourney : Maybe Bool
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)